### DISCUSSION TITLE
[design:suggest] Lack of space in "Create Chart" GUI

### DISCUSSION THREAD

**Discussion by goldjee** [apache/superset#32799] [Ideas]
## Screenshot

<img width="1440" alt="Снимок экрана 2024-05-31 в 22 47 15" src="https://github.com/apache/superset/assets/7242338/24f9434c-7bd5-41d9-9ea2-a51283fcde73">

## Description

There are a lot of cases when a chart has quite a lot of tags and a long description. When you select a chart in the gallery, an info panel appears and eats a significant chunk of space from the gallery itself. On a 13'' display there will be too little space left to browse other charts or to read the description comfortably. Especially it affects localizations for languages that are not as brief as English.

## Design input

I suggest to review the layout of "Create chart" screen and probably take advantage of vertical scroll for the whole page.

**Comment by rusackas** [comment]
I remember that we tried vertical scrolling, but having the "Create New Chart" button being offscreen caused even more significant problems. Hopefully @kasiazjc has some ideas.

**Comment by goldjee** [comment]
I've been thinking about this for a while, and it seems we could get rid of this page altogether. Full chart editor already has the UI allowing to choose dataset and visualization type. When user hits "+ chart" button we could take them there and highlight the dataset selector.

<img width="660" alt="image" src="https://github.com/apache/superset/assets/7242338/9114a400-7e18-482e-9da6-2acdf6048b29">

As for chart type, we could pick flat table as the default. If another chart type is desired, user can always change it.

<img width="1136" alt="image" src="https://github.com/apache/superset/assets/7242338/4f2e345a-8317-4388-9b44-2da19dd3b6df">

This modal window has a bit more space here and provides the same functionality.

**Comment by rusackas** [comment]
Moving this to an Ideas thread, since that's likely more fitting than a bug report. Perhaps @kasiazjc has some ideas here. Any interest in contributing if folks agree on your proposal?
